planner-api/src/main/java/pl/edu/agh/kuce/planner/event/EventController.java

```diff
@@ -36,4 +40,18 @@ public OneTimeEventData create(@Valid @RequestBody final OneTimeEventDataInput d
     public ListResponse list(@Current final User user) {
         return eventService.list(user);
     }
+
+    @PutMapping("/{id}")
```

Comment:
In openapi.yaml you wrote that this endpoint returns 201 CREATED, but you didn't specify it here, so it returns 200 OK. I would consider changing it to 204 NO_CONTENT, because it doesn't really create a new Event. It replaces the old one, but does not change its ID. Therefore, frontend doesn't need a response body.